src/components/Assemblies/CabinetRole.svelte

```diff
@@ -1,21 +1,56 @@
 <script lang="ts">
-	import type { CabinetSeat } from './shared';
+	import type { AssemblyMember } from '$lib/datasheets/assembly-member';
+	import type { CabinetSeat, TooltipProp } from './shared';
 
 	export let cabinets: CabinetSeat[];
 	export let role: CabinetSeat['role'];
+
+	let tooltipProp: TooltipProp | null = null;
+
+	const showTooltip = (event: MouseEvent, member: AssemblyMember) => {
+		const name = `${member.firstname} ${member.lastname}`;
+
+		tooltipProp = {
+			title: name,
+			additional: member.assemblyRole.role,
+			x: event.layerX,
+			y: event.layerY + 20
+		};
+	};
+
+	const hideTooltip = () => {
+		tooltipProp = null;
+	};
 </script>
 
 <div class="role">
 	<p class="heading-compact-01">{role}</p>
 	<div class="group-dot">
 		{#each cabinets.find((c) => role === c.role)?.parties || [] as party}
-			{#each Array.from({ length: party.count || 0 }, (_, i) => i) as _}
-				<span class="dot" style="background-color: {party.color || '#8D8D8D'};"></span>
+			{#each party?.members ?? [] as member}
+				<div
+					role="tooltip"
+					class="dot"
+					style="background-color: {party.color || '#8D8D8D'};"
+					on:mouseenter={(e) => showTooltip(e, member)}
+					on:mouseleave={hideTooltip}
+				></div>
 			{/each}
 		{/each}
 	</div>
 </div>
 
+<div
+	style:left={`${tooltipProp?.x ?? 0}px`}
+	style:top={`${tooltipProp?.y ?? 0}px`}
```

Comment:
No need to use string template literature because Svelte allows the expression `{}` in the attribute. You can do
```svelte
<div
	style:left="{tooltipProp?.x ?? 0}px"
	style:top="{tooltipProp?.y ?? 0}px"
```
which is easier to read

Reply:
Thanks, I've learned new thing everyday.